packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.Error.Example.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { ChartDataMode, HorizontalBarChart, IHorizontalBarChartProps } from '@fluentui/react-charting';
+
+interface IHorizontalBarChartState {
+  chartMode: ChartDataMode;
+}
+
+export class HorizontalBarChartErrorExample extends React.Component<
```

Comment:
This is empty data example not error example

Reply:
updated